Jenkinsfile

```diff
@@ -32,10 +32,5 @@ try {
 catch (Exception e) {
     echo 'Failure encountered'
 
-    node("ec2-stateful") {
```

Comment:
will this have a negative effect for successful builds (i.e. carrying over state that we don't want to across consecutive builds that run on the same instance)?